src/vivarium/framework/engine.py

```diff
@@ -237,7 +237,11 @@ def step(self) -> None:
         self._logger.debug(self._clock.time)
         for event in self.time_step_events:
             self._lifecycle.set_state(event)
-            self.time_step_emitters[event](self._population.get_population(True).index)
+            pop_to_update = self._clock.aligned_pop(
```

Comment:
I don't love the name `aligned_pop`. What about something like `get_active_index` or something like that?

Reply:
renamed to `get_active_population`